lib/src/rows/Rows.stories.tsx

```diff
@@ -0,0 +1,314 @@
+import React from "react";
+import Title from "../../.storybook/components/Title";
+import styled from "styled-components";
+import DxcRows from "./Rows";
+
+export default {
+  title: "Rows",
+  component: DxcRows,
+};
+
+export const Chromatic = () => (
+  <>
+    <Title title="Default" theme="light" level={4} />
+    <Container>
+      <DxcRows>
+        <DxcRows.Row>
+          <Placeholder width="small" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="medium" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="large" />
+        </DxcRows.Row>
+      </DxcRows>
+    </Container>
+    <Title title="AlignX = end" theme="light" level={4} />
+    <Container>
+      <DxcRows alignX="end">
+        <DxcRows.Row>
+          <Placeholder width="small" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="medium" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="large" />
+        </DxcRows.Row>
+      </DxcRows>
+    </Container>
+    <Title title="AlignX = center" theme="light" level={4} />
+    <Container>
+      <DxcRows alignX="center">
+        <DxcRows.Row>
+          <Placeholder width="small" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="medium" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="large" />
+        </DxcRows.Row>
+      </DxcRows>
+    </Container>
+    <Title title="AlignX = baseline" theme="light" level={4} />
+    <Container>
+      <DxcRows alignX="baseline">
+        <DxcRows.Row>
+          <Placeholder width="small" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="medium" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="large" />
+        </DxcRows.Row>
+      </DxcRows>
+    </Container>
+    <Title title="AlignX = stretch" theme="light" level={4} />
```

Comment:
This should be AlignX = start example instead of stretch.

Reply:
Done!